package-lock.json

```diff
@@ -17,10 +17,10 @@
       "dev": true,
       "optional": true,
       "requires": {
-        "co": "4.6.0",
```

Comment:
hi! could you please undo all the changes to the package-lock file? / not commit the changes?

Reply:
sure, doing it now